### DISCUSSION TITLE
[TROUBLESHOOTING] Program properties won't show properly in 22000.1

### DISCUSSION THREAD

**Discussion by SamuraisEpic** [valinet/ExplorerPatcher#942] [General]
Hello, i wanted to know if there was a fix for the properties window which doesn't show up at all, instead showing a thin darkened bar, when i click on it in the taskbar. for background info, I'm running 22000.1, and using it on my main build. it takes some weird stuff to install the newer versions and even v22000.1.05, and on top of that, when I open properties to change the programs settings, it shows the icon on my taskbar, but only a thin dark line on the display. nothing happens when I click on it in my Taskbar but hovering over the icon to show the preview seems to be working. (it'll show the program's settings, but no background in the Taskbar preview.)

**Comment by Poopooracoocoo** [comment]
Just to clarify, what build of Windows and EP are you running? Are they both up to date?

**Reply by SamuraisEpic** [reply]
I'm running Windows 10 ("11") build 10.0.22000.1 (I won't update to full 11 because it sucks.) With the latest preview version of EP (I don't know the version number).

**Reply by Poopooracoocoo** [reply]
EP enables Mica when the Windows build number >= 21996. It could be getting tripped up because you're on the version right before W11 and Mica. On builds before 21996, EP's properties doesn't use Mica. 

Just a guess. EP doesn't officially support that build so idk. Why stay on that build of Windows by the way?

**Reply by SamuraisEpic** [reply]
The reason I stay on 22000.1 is because I personally feel like it's a lot better than the "improved experience" that windows 11 supposedly offers. Now. I think that some of the first release builds for explorerpatcher were for 22000.1, and so I thought it'd be fine. Now I don't know whether there's a way to get mica on these older builds, but id like a version without Mica. Is there a way to manually change settings from regedit or disable Mica?

**Reply by valinet** [reply]
They were fine, but no one tested after that. I don’t remember well, but if Mica was only introduced on 22000.65, then I can make sure to only enable it then. Although, yeah, idk, I haven’t tested the app on that build in a very long time, so idk what to say. With EP, newer builds get you mostly the same things as 22000.1. I also stayed on that initially, but moved on once I got things going with EP, that was the whole purpose of it.

**Reply by SamuraisEpic** [reply]
Ok, would you be able to add a check that like you said, disables Mica below 22000.65 for the next stable and preview builds?

**Reply by valinet** [reply]
Ofc, I will fix it, it's just that I can't work on everything at once.

**Reply by valinet** [reply]
Tbh, supporting all these builds is a pain. Now I have to load up a VM with 22000.1 and test everything in there...

**Reply by SamuraisEpic** [reply]
I understand, and I'm sorry. I know that devs including yourself are just mere mortals, but it's really amazing that you keep up with these changes rather quickly. and I am eternally grateful for that.

**Reply by valinet** [reply]
Yeah, no need to be sorry, don't worry, it's fine, I guess. Thanks for the good words.

**Reply by valinet** [reply]
Yeah, we have to reclassify this. It's actually not a bug with ExplorerPatcher. I remember it from last summer. It's actually a bug in Windows' compositor (Desktop Window Manager) that happens on 22000.1 and .51 (maybe?) and it was fixed in some later version. It would be good to determine what that version is (maybe .65?).

The bug manifests in the fact that all windows that use some form of `DwmExtendFrameIntoClientArea` are crapped out. That's what happens to the Properties window there as well. It happens with the Windows Photo Viewer as well, you'll see, if you open that, the bottom area (which would be glass if Windows Aero would still be a thing - has `DwmExtendFrameIntoClientArea` applied to it) won't be visible. So yeah, it's a bug in `dwm.exe`.

Pffff...

**Reply by valinet** [reply]
Anyway, is EP actually working on 22000.1 still? Like, the Properties window can be fixed, but is the program working on that build?

**Reply by SamuraisEpic** [reply]
Yes, but it takes some sketchy steps to install. You first need to install the version you want, then install v22000.1.05, then uninstall v22000.1.05, and then run the command to uninstall v22000.1.05 again. That should leave you with the taskbar from the version that you initially installed. Apart from the Mica window and the install process, the rest of the program seems to work fine.

**Reply by SamuraisEpic** [reply]
Just curious, has the issue been fixed?

**Reply by valinet** [reply]
I think so, can't you test it?

**Reply by SamuraisEpic** [reply]
Ok, I'll see to test it when I'm able to. Btw, is this fix in both future stable and preview builds?

**Reply by Poopooracoocoo** [reply]
@SamuraisEpic check the preview build

**Reply by valinet** [reply]
I checked it now, it seems to be fixed for now.

**Reply by SamuraisEpic** [reply]
Ok, slight issue. So I don't know if this is because 22000.1 supports 10SM natively, but after installing the program, it removed all my win32 shortcuts. The folder itself seems to be intact, but the programs aren't showing on the menu. Only UWP apps are
